app/common/auth/login.directive.js

```diff
@@ -61,6 +63,19 @@ function LoginController(
         $scope.failed = false;
         $scope.processing = false;
         $scope.$parent.closeModal();
+        if ($scope.email === 'admin') {
+            redirectAdminEmailToAccountSettings();
+        }
+    }
+
+    /**
+     * redirectAdminEmailToAccountSettings is called when there is a login with the 'admin' email instead of a proper email.
+     * This is part of an effort to force users to have proper emails and not use the default email/password combination that the
+     * system had during the setup process.
+     * references https://github.com/ushahidi/platform/issues/1714
+     */
+    function redirectAdminEmailToAccountSettings() {
+        ModalService.openTemplate('<account-settings admin-user-setup="true"></account-settings>', 'Change your email and password', false, false, false, false);
```

Comment:
I wonder if it might be better to just wrap the user-profile by itself in a new directive ie `adminUserSetup` rather than sprinkling the code with conditionals based on `adminUserSetup` ??

Reply:
yup, that makes more sense... 
thanks!